src/_common/community/community.model.ts

```diff
@@ -59,6 +59,11 @@ export async function $leaveCommunity(community: Community) {
 	}
 }
 
+export enum CommunityPresetChannelType {
```

Comment:
We've never used enums in our project because they add in extra imports/runtime code. It looks like we can use const enums now which replace the values at the call site during compilation. Let's do that here?